spec/acceptance/nodesets/default.yml

```diff
@@ -1,6 +1,19 @@
 HOSTS:
```

Comment:
We usually purge this file within modulesync and create nodesets on demand with https://github.com/puppetlabs/beaker-hostgenerator

Reply:
@bastelfreak The tests are going to quickly get more complicated than this will handle so I'd rather not right now.

Reply:
To clarify, these will be multi-node in the near future. Via work on `simp-iptables`, we have found that this is the only way that you can properly uncover bugs in the underlying system.